docs/source/user_guide_advanced/multi_gpu_training.rst

```diff
@@ -0,0 +1,75 @@
+.. _multi_gpu_training:
+
+###################
+Multi-GPU Training
+###################
+
+Multi-GPU training allows you to distribute the load of model training across GPUs.
+This helps overcome OOMs in addition to accelerating training.
+
+To use this feature, set :ref:`num_gpus <config_num_gpus>` in your config file.
+
+How to choose batch_size
+========================
+
+Multi-GPU training distributes batches across multiple GPUs in a way that maintains the same
+effective batch size as if you ran on 1 GPU. **Thus, if you reduced batch size in order to make
+your model fit in one GPU, you should increase it back to your desired effective batch size.**
+
+The batch size configuration parameters that this applies to are ``training.train_batch_size`` and
+``training.val_batch_size`` for the labeled frames, and ``dali.train.base.sequence_length`` and
+``dali.train.context.batch_size`` for unlabeled video frames. Test batch sizes are not relevant
+to this document as testing only occurs on one GPU.
+
+Calculate of per-GPU batch size
+-------------------------------
+
+Given the above, you need not worry about how lightning-pose calculates per-GPU batch size,
```

Comment:
split this into labeled and unlabeled batches; what you're describing here are the unlabeled batch sizes but that's not immediately clear